src/example-relay/src/Homepage/locations/Location.js

```diff
@@ -20,12 +18,11 @@ function Location({ location }: Props) {
   const name = location.name ?? '';
   return (
     <li>
-      <Stack direction="row" spacing="compact">
-        <CountryFlag location={location} />
```

Comment:
I believe there was something like `iconUrl` which you could use here. But no big deal.

Reply:
Ahh, yes, we have a `countryFlagURL`, I see if I can re add it in a later PR